### DISCUSSION TITLE
Student Developer Pack rejected

### DISCUSSION THREAD

**Discussion by valentinarau** [community/community#51377] [GitHub Education]
Hi, i need help to renew my Student Developer Pack. 
I have completed my profile with my full name as it is registered at my university and I provided a short description. I have uploaded a Certificate of Registered Student Status, a schedule of my classes and the certificate of enrollment. All those had visible my student ID, my name and the current date. However, it didn't seemed to work, because I submitted 3 to 4 times the information but it was rejected immediately.

**Comment by samus-aran** [comment]
Hi @valentinarau 👋 

Happy you're here!

Here are a few tips to help students re-evaluate their application.

1. Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

2. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Hope these help 👍
